src/references/index.ts

```diff
@@ -48,6 +48,24 @@ export {
   WYRE_SUPPORTED_COUNTRIES_ISO,
 } from './wyre';
 
+export {
+  ENSRegistryWithFallbackABI,
+  ENSETHRegistrarControllerABI,
+  ENSReverseRegistrarABI,
+  ENSBaseRegistrarImplementationABI,
+  ENSPublicResolverABI,
+} from './ens';
+
+export const ensRegistryAddress = '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e';
```

Comment:
would be good to export these ENS related addresses from `references/ens/index` too